### DISCUSSION TITLE
Blocking "Welcome to ***" message

### DISCUSSION THREAD

**Discussion by pincette** [NadyaNayme/TidyChat#77] [Q&A] [answered]
Hello everyone!
Is there way to block welcome messages when you travel between worlds? I mean, Welcome to FINAL FANTASY XIV! and "Welcome to _servername_" messages?

**Comment by NadyaNayme** [comment]
If I recall correctly those are Announcement channel messages and Tidy Chat has a policy to never touch that channel (or the Gamemaster channels) because it could be really problematic if an important message finds itself being blocked as a false positive. If they are - there is no way to block them with Tidy Chat.

I could be wrong and those are System channel messages. If you add a custom filter for "Welcome to" and for System-chanel messages it will block them if they are.

**Reply by pincette** [reply]
Apparently, the System custom filter doesn't work in this case.

**Reply by NadyaNayme** [reply, marked as answer]
That would mean they are a part of the Announcement channel after all. I don't plan to support blocking messages (even if it's user opt-in w/ custom filters) for that channel. For every responsible user who understands its their fault they didn't see an emergency maintenance announcement because they accidentally wrote a filter that blocks all Announcements there are 1000 people who'd complain they missed a super important message (even though they'd be the ones to blame for that).
